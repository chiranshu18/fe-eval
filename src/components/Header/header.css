.header{
    background-color: black;
    padding-left: 10vw;
    padding-right: 10vw;
    padding-top: 30px;
    padding-bottom: 30px;
}

.header-text{
    font-size: 70px;
    color: white;
    font-family: 'Big Shoulders Inline Display', cursive;
}