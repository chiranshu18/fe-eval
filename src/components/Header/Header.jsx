import React from 'react'
import './header.css'

const Header = () => {
  return (
    <div className='header'>
        <h1 className='header-text'>EVENTIFY</h1>
    </div>
  )
}

export default Header