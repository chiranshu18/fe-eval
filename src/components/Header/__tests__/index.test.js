import Header from '../Header'
import { render , screen} from "@testing-library/react";

describe('header', () => {
    
    it('should display heading when rendered', () => {
        render(<Header />);
        expect(screen.getByText('EVENTIFY')).toBeTruthy();
    });

    it('Should render correctly', () => {
        const {asFragment} = render(<Header />)
        expect(asFragment()).toMatchSnapshot();
    })
});