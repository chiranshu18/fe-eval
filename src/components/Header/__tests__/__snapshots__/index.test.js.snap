// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1
      class="header-text"
    >
      EVENTIFY
    </h1>
  </div>
</DocumentFragment>
`;

exports[`header Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1
      class="header-text"
    >
      EVENTIFY
    </h1>
  </div>
</DocumentFragment>
`;
